build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts

```diff
@@ -136,9 +137,10 @@ fun isExpectedCompileCacheMiss() =
     isInBuild(
         "Check_CompileAllBuild",
         "Component_GradlePlugin_Performance_PerformanceLatestMaster",
-        "Component_GradlePlugin_Performance_PerformanceLatestReleased",
-        "Check_Gradleception"
-    )
+        "Component_GradlePlugin_Performance_PerformanceLatestReleased"
+    ) || (isInBuild("Check_Gradleception") && !isOnBranch("master", "release"))
```

Comment:
I am not sure we should change something here. I think we are not using the `CACHE_MISS` tag at all, right? We probably should monitor it at some point.